components/catalogs/front/src/components/ProductMapping/components/SourceParameters/DefaultValue.tsx

```diff
@@ -46,6 +46,18 @@ export const DefaultValue: FC<Props> = ({source, onChange, error, targetTypeKey}
                 />
             );
             break;
+        case 'number':
+            element = (
+                <NumberInput
+                    data-testid={'number-default-value'}
+                    onChange={value => onChangeMiddleware({...source, default: value})}
+                    placeholder={translate(
+                        'akeneo_catalogs.product_mapping.source.parameters.default_value.placeholder'
+                    )}
+                    value={source.default !== undefined && source.default !== null ? source.default.toString() : ''}
```

Comment:
can we not do the same tests as other inputs ? 
```suggestion
                    value={typeof source.default === 'number' ? source.default.toString() : ''}
```